tests/lora/tpu/test_pallas_kernels.py

```diff
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+import torch
+
+# Required to register the custom ops
+import vllm.lora.ops.xla_ops.pallas  # noqa # pylint: disable=unused-import
+
+N_TOKENS = [
+    8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536,
+    131072
+]
+HIDDEN_SIZES = [128, 256, 512, 896, 1024, 2048, 4096, 8192, 8320]
+
+DTYPES = [torch.float16, torch.bfloat16]
```

Comment:
torch.bfloat16 and torch.float32 seems more common

Reply:
Yep that makes sense, I'm also going to reduce the number of test cases since this one takes forever to run

Reply:
Ah actually torch.float32 isn't supported rn - https://github.com/vllm-project/vllm/blob/main/vllm/platforms/tpu.py#L94